frontend/src/Components/ResultBox.js

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components/macro'
+import { Typography } from '@material-ui/core'
+import { saveAnswers } from '../Services/api-service'
+
+export default function ResultBox({
+  question,
+  givenAnswer,
+  correctAnswer,
+  user,
+  token,
+}) {
+  const isRight = givenAnswer === correctAnswer
+  const pointsMade = () => {
+    if (isRight) {
+      return 1
+    } else {
+      return 0
+    }
+  }
+  const answer = {
+    userName: user.userName,
+    question: question,
+    score: pointsMade(),
+  }
+  saveAnswers(token, answer).catch(e => console.log(e))
```

Comment:
```
const [isRight, setIsRight] = useState(false)

useEffekt(() => {
if(givenAnswer === correctAnwser){
setIsRight(true)
const answer = {
{
    userName: user.userName,
    question: question,
    score: 1
  }
saveAnswers(token, answer).catch(e => console.log(e))
}
```suggestion
  const isRight = givenAnswer === correctAnswer
  const pointsMade = () => {
    if (isRight) {
      return 1
    } else {
      return 0
    }
  }
  const answer = {
    userName: user.userName,
    question: question,
    score: pointsMade(),
  }
  saveAnswers(token, answer).catch(e => console.log(e))
},[])
```